arduino/WandCore/LEDClient.cpp

```diff
@@ -10,6 +10,10 @@
 #define CHAN_1 1
 #define CHAN_2 2
 
+// RGB Colorloop state vars
+int RGBColorloopSection = 0; // this holds the state of where in the 6 rainbow sections (0-5) the colorloop is

```

Comment:
Nit-picky thing, usually in camelCase conventional languages, even initialisms get lowercased if they're the first word, so this would be `rgbColorloopSection`.